### PR TITLE
WIP: initial version: Nordic.js post

### PR DESCRIPTION
**Pull request by TimKam**

### CODE REVIEW

**Comment by hilton** [comment]
Nice write-up. One thing that would make it a lot better would be if you could add one sentence for each talk to say why you liked it. Also, one of the talks doesn't have any comment at all.

**Comment by TimKam** [comment]
Thanks for the feedback! Jonas will contribute as well, hence the missing parts :-)

File: _posts/2017-09-13-nordic-js.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+[Nordic.js](http://nordicjs.com/) is a JavaScript conference that takes place annually (since 2014) in Stockholm, Sweden.
+We attended the 2017 conference; here's our perspective on the event.
+
+## About the conference
+The conference took place over two days (Thursday and Friday), which inspired us to combine it with a Stockholm sightseeing weekend ;-)
+JavaScript celebrity [MPJ](https://twitter.com/mpjme) co-moderated the conference alongside with [Unn Swanström](https://twitter.com/unnderbar).
```
Comment (johnny353535) [inline _posts/2017-09-13-nordic-js.md]: I think the `with` can be axed

File: _posts/2017-09-13-nordic-js.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+As there was only one track, disciplined visitors could catch all talks.
+This year, Nordic.js was located in a former fabric hall in Stockholm's old industrial area *Sickla*.
```
Comment (johnny353535) [inline _posts/2017-09-13-nordic-js.md]: I'd call it "warehouse" instead of "fabric hall"

File: _posts/2017-09-13-nordic-js.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+The venue helped create a festival-like atmosphere and inspire creative thought.
+The conference featured a broad range of talks with topics ranging from *accessibility* over *ECMAScript standardization* and *inclusion* to *web technology evolution*.
+
+
+## Notable talks
+While we genuinely enjoyed all talks, the following selection stroke us as particularly noteworthy.
+
+### Harriet Lawrence on sociolinguistics in tech communications
+In her talk *Sociolinguistics and the Javascript community: a love story*, Harriet elaborated on the importance of careful and explicit communications.
+While Harriet's talk seemed to be targeted at open source contributors and maintainers, its essence applies to all software development projects and especially large-scale project that involve large and geographically distributed groups of people, who might not be aware of each other's cultural backgrounds and communication preferences.
```
Comment (johnny353535) [inline _posts/2017-09-13-nordic-js.md]: I'd end the sentence after "that" and start the following sentence with "These projects...".
There's also an "s" missing.

File: _posts/2017-09-13-nordic-js.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+### Léonie Watson on accessibility
+*You're only supposed to blow the bloody doors off!*
+
+You find Léonie's slides at [http://decks.tink.uk/2017/frontendunited/index.html](http://decks.tink.uk/2017/frontendunited/index.html).
+
+### Paul Kinlan on the future of the web
+*The future of the web and the curious case of the unintended silo*
+
+### Claudia Hernández on JavaScript without alphanumerics
+Claudia's talk *Down the Rabbit Hole: JS in Wonderland* about curious JavaScript intricacies was arguably the most entertaining presentation of the conference.
```
Comment (johnny353535) [inline _posts/2017-09-13-nordic-js.md]: I'd replace "intricacies" with "quirks" and remove "curious"

**Comment by TimKam** [comment]
@johnny353535 Agreed with your feedback. Can you just add the changes when you commit "your" content? :-)
Will work in Peter's feedback ("Why did I like the talks?") later.

File: _posts/2017-09-19-nordic-js.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+This is just one example of the many ways that ARIA helps to make website more accessible.
+You can find more examples in Léonie's slides at [http://decks.tink.uk/2017/frontendunited/index.html](http://decks.tink.uk/2017/frontendunited/index.html).
+
+### Claudia Hernández on JavaScript without alphanumerics
+Claudia's talk *Down the Rabbit Hole: JS in Wonderland* about JavaScript's quirks was arguably the most entertaining presentation of the conference.
+The Dailymotion engineer presented a six-character (``(![]+)``) subset of JavaScript (as presented on [this website](http://www.jsfuck.com/)) that allows running any program by creatively exploiting JavaScript's lack of type safety.
+For example, [this JSFiddle](https://jsfiddle.net/7uph4tsd/) contains the ``(![]+)``-equivalent of ``alert('Signavio')``.
+You find (a 2016 version of) Claudia's slides [here on Speaker Deck](https://speakerdeck.com/claudiahdz/down-the-rabbit-hole-javascript-in-wonderland).
+
+**I liked this talk because…**
```
Comment (TimKam) [inline _posts/2017-09-19-nordic-js.md]: @johnny353535 Could you also add such a "I liked because" sentence to your talk?

**Comment by TimKam** [comment]
@rstiles Can you take a look at the language?

File: _posts/2017-09-19-nordic-js.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+[Nordic.js](http://nordicjs.com/) is a JavaScript conference that takes place annually (since 2014) in Stockholm, Sweden.
+We attended the 2017 conference; here's our perspective on the event.
+
+## About the conference
+The conference took place over two days (Thursday and Friday), which inspired us to combine it with a Stockholm sightseeing weekend ;-)
+JavaScript celebrity [MPJ](https://twitter.com/mpjme) co-moderated the conference alongside [Unn Swanström](https://twitter.com/unnderbar).
+As there was only one track, disciplined visitors could catch all talks.
+This year, Nordic.js was located in a former fabric hall in Stockholm's old industrial area *Sickla*.
+The venue helped create a festival-like atmosphere and inspire creative thought.
```
Comment (rstiles) [inline _posts/2017-09-19-nordic-js.md]: "inspireD creative thought"

**Pull request merged**